packages/fluentui/docs/src/components/ComponentDoc/ComponentPerfExample/ComponentPerfChart.tsx

```diff
@@ -6,7 +6,9 @@ const ExtremesFilter = ({ onChange }) => (
   <Checkbox label="Show extremes" onChange={(e, { checked }) => onChange({ extremes: checked })} />
 );
 
-export const ComponentPerfChart = ({ perfTestName }) => {
+const ComponentPerfChart = ({ perfTestName }) => {
```

Comment:
Thought you were moving away from default exports?

Reply:
Yup, good catch